scripts/us_census/pep/pep_by_asrh/preprocess.py

```diff
@@ -0,0 +1,1072 @@
+    """
+    if age_grp == 0:
+        return "0"
+    if age_grp == 1:
+        return "1To4"
+    if age_grp == 18:
+        return "85OrMore"
+    start = 5 * (age_grp - 1)
+    end = 5 * (age_grp) - 1
+    return f"{start}To{end}"
+
+
+def _add_measurement_method(data_df: pd.DataFrame, src_col: str,
+                            tgt_col: str) -> pd.DataFrame:
+    """Adds Measurement Method either CensusPEPSurvey or
+    dcAggregate/CensusPEPSurvey to tgt_col column.
+    Args:
+        data_df (pd.DataFrame): Input DataFrame
+        src_col (str): SV Column Name
+        tgt_col (str): Measurement Method Column
+
+    Returns:
+        pd.DataFrame: DataFrame with New Columns
+    """
+    data_df[tgt_col] = data_df[src_col].str.split("_").str[-1]
+    data_df[tgt_col] = data_df[tgt_col].str.replace(r"^(?!computed).*",
+                                                    "dcs:CensusPEPSurvey",
+                                                    regex=True)
+    data_df[tgt_col] = data_df[tgt_col].str.replace(
+        "computed", "dcs:dcAggregate/CensusPEPSurvey", regex=False)
+
+    # Dervied SV"s has "_computed" as part of the name,
+    # to differentiate them with source generated SV"s
+    # Removing "_computed" in the SV"s names.
+    data_df[src_col] = data_df[src_col].str.replace("_computed", "")
+    return data_df
+
+
+def _load_data_df(path: str,
+                  file_format: str,
+                  header: str = None,
+                  skip_rows: int = None,
+                  encoding: str = "UTF-8") -> pd.DataFrame:
+    """
+    Returns the DataFrame using input path and config.
+    Args:
+        path (str): Input File Path
+        file_format (str): Input File Format [csv, txt, xls, xlsx]
+        header (str, optional): Input Dataset Header Row Line Number.
+        DataFrame will consider header value and make it as header.
+        Defaults to None.
+        skip_rows (int, optional): Skip Rows Value for txt files.
+        This is helpful to skip initial rows in txt file.Defaults to None.
+        encoding (str, optional): Input File Encoding while
+        loading data into DataFrame.Defaults to None.
+
+    Returns:
+        data_df (pd.DataFrame): Dataframe of input file
+    """
+    data_df = None
+    if file_format.lower() == "csv":
+        data_df = pd.read_csv(path, header=header, encoding=encoding)
+    elif file_format.lower() == "txt":
+        data_df = pd.read_table(path,
+                                index_col=False,
+                                delim_whitespace=True,
+                                engine="python",
+                                header=header,
+                                skiprows=skip_rows,
+                                encoding=encoding)
+    elif file_format.lower() in ["xls", "xlsx"]:
+        data_df = pd.read_excel(path, header=header)
+    data_df = _convert_to_int(data_df)
+    return data_df
+
+
+def _create_sv(desc: str, age: str) -> str:
```

Comment:
This function name should be more descriptive. It is specific to the age SVs so perhaps something like _create_sv_age() is better?